Automattic/vip-go-mu-plugins
# Search: Only ensure index existence for single-sites

## Description
We should not be creating new indexes via on `ensure_index_existence()` for multisites. This can result in unused indexes which may lead to resource consumption.

## Changelog Description

### Plugin Updated: Enterprise Search

Only ensure index existence for single-sites.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Create a multisite ES env
2) Create a new subsite
3) Perform a search
4) Check `wp vip-search get-indexes` to ensure that an index has not been created
